docs/application/dotnet/guides/text-input/voice-control-manager.md

```diff
@@ -0,0 +1,1099 @@
+
+    You can use the `RecognitionModeType` enumeration of the [Tizen.Uix.VoiceControlManager.RecognitionModeType](https://samsung.github.io/TizenFX/latest/api/Tizen.Uix.VoiceControlManager.RecognitionModeType.html) enumeration as a parameter.
+
+    The default value of `RecognitionModeType` is `RecognitionModeType.StopBySilence`. If you want to set the manual mode, you can use `RecognitionModeType.Manual`.
+
+    This function is used when the voice control manager is in the `Ready` state:
+
+    ```csharp
+    void GetSetRecognitionMode()
+    {
+        try
+        {
+            /// Get
+            RecognitionModeType recognitionModeType = VoiceControlManagerClient.RecognitionMode;
+
+            /// Set
+            VoiceControlManagerClient.RecognitionMode = recognitionModeType;
+        }
+        catch (Exception e)
+        {
+            /// Error handling
+        }
+    }
+    ```
+
+-   Get or set private data between voice control manager and voice control engine.
+
+    You can use the `GetPrivateData()` and `SetPrivateData()` methods of the [Tizen.Uix.VoiceControlManager.VoiceControlManagerClient](https://samsung.github.io/TizenFX/latest/api/Tizen.Uix.VoiceControlManager.VoiceControlManagerClient.html) class.
+
+    `GetPrivateData()` is used when the parameters move from voice control engine to voice control manager, while `SetPrivateData()` is used when the parameters move from voice control manager to voice control engine.
+
+    This option must be set when the voice control manager is in the `Ready` state:
+
+    ```csharp
+    void GetSetPrivateData()
+    {
+        try
+        {
+            /// Get
+            string privateData = VoiceControlManagerClient.GetPrivateData("privateKey");
+
+            /// Set
+            VoiceControlManagerClient.SetPrivateData("privateKey", "privateData");
+        }
+        catch (Exception e)
+        {
+            /// Error handling
+        }
+    }
+    ```
+
+
+
+<a name="commands"></a>
+## Manage Commands
+
+You can use command group to manage the commands. You can add or remove the commands to the command group and retrieve the command information using the command group.
+
+To create a command group and commands:
+
+1.  Create a command group with a command group handle.
+
+    The command group can have multiple commands. Each command has a text and a type.
+
+    The group can have `Background`, `Exclusive`, `Foreground`, `System`, `SystemBackground`, and `Widget` type commands.
+
+    For more information on the command types, see [Tizen.Uix.VoiceControlManager.CommandType](https://samsung.github.io/TizenFX/latest/api/Tizen.Uix.VoiceControlManager.CommandType.html) enumeration.
+
+    > [!NOTE]
+    > The order of command group priority is `System`, `Widget`, `Foreground`, `SystemBackground` and `Background`. The `Exclusive` is used on special situation.
+    > Normally, The `System` and `SystemBackground` commands is only registered in voice control manager.
```

Comment:
Please replace "are" with "is" and write "the" in sentence form. Kindly write the sentence as:

Normally, the `System` and `SystemBackground` commands are only registered in the voice control manager.